hail/src/main/scala/is/hail/rvd/UnpartitionedRVD.scala

```diff
@@ -14,20 +14,17 @@ object UnpartitionedRVD {
     UnpartitionedRVD(rowType, ContextRDD.empty[RVDContext, RegionValue](sc))
 
   def apply(rowType: TStruct, crdd: ContextRDD[RVDContext, RegionValue]): RVD = {
-    new UnpartitionedRVD(rowType, crdd)
+    OrderedRVD.unkeyed(rowType, crdd).toOldStyleRVD
   }
 }
 
 class UnpartitionedRVD private (val rowType: TStruct, val crdd: ContextRDD[RVDContext, RegionValue]) extends RVD {
   self =>
 
-  override def toOrderedRVD: OrderedRVD = {
-    new OrderedRVD(
-      typ = OrderedRVDType(FastIndexedSeq.empty, rowType),
-      partitioner = OrderedRVDPartitioner.unkeyed(crdd.getNumPartitions),
-      crdd = crdd
-    )
-  }
+  assert(false)
```

Comment:
this shouldn't be here, right? Can still get unpartitioned RVDs from legacy files, right?

Reply:
This PR is to allow me to delete UnpartitionedRVD completely, which I just did in the followup branch. So I wanted to be sure I got rid of all uses.

We should leave in `UnpartitionedRVDSpec`, but read it in as an `OrderedRVD` with empty key. This should have identical semantics.